packages/mui-base/src/TablePaginationUnstyled/TablePaginationUnstyled.types.ts

```diff
@@ -12,7 +13,8 @@ export interface LabelDisplayedRowsArgs {
 export type ItemAriaLabelType = 'first' | 'last' | 'next' | 'previous';
 
 export interface TablePaginationUnstyledRootSlotPropsOverrides {}
-export interface TablePaginationUnstyledActionsSlotPropsOverrides {}
+export interface TablePaginationUnstyledActionsSlotPropsOverrides
```

Comment:
The Overrides interface should be empty (by convention). If you'd like to specify the type of the `slotProps` field, define slotProps.actions as `actions?: SlotComponentProps<typeof TablePaginationActionsUnstyled, TablePaginationUnstyledActionsSlotPropsOverrides, TablePaginationUnstyledOwnerState>`, similarly to how SelectUnstyled defines its slotProps.popper.